src/RequestHandler/outofprocess/forwardinghandler.cpp

```diff
@@ -330,7 +330,7 @@ FORWARDING_HANDLER::OnExecuteRequestHandler()
     {
         pResponse->SetStatus(400, "Bad Request", 0, hr);
     }
-    else
+    else if (fFailedToStartKestrel && !m_pApplication->QueryConfig()->QueryDisableStartUpErrorPage())
```

Comment:
why do we need to check if we failed to start kestrel here?

Reply:
this is a flag indicating we failed to get process (for out-of-process case)